src/sections/create-dataset/CreateDatasetFactory.tsx

```diff
@@ -0,0 +1,11 @@
+import { CreateDatasetForm } from './CreateDatasetForm'
+import { ReactElement } from 'react'
+export function CreateDatasetFormPresenter() {
+  return <CreateDatasetForm />
+}
+
+export class CreateDatasetFactory {
+  static create(): ReactElement {
+    return <CreateDatasetFormPresenter />
+  }
+}
```

Comment:
The recent changes look good, I want to approve the PR already. I would just remove this `CreateDatasetFormPresenter` component because it seems redundant

```suggestion
export class CreateDatasetFactory {
  static create(): ReactElement {
    return <CreateDatasetForm />
  }
}
```

Reply:
Thanks Melina, this was a leftover that should have been refactored earlier. It has now been removed!